### PR TITLE
RDAPI-21725 update documentation on yamles externalization files

### PR DESCRIPTION
**Pull request by dominiquemongelli**
https://deploy-preview-1433--axway-open-docs.netlify.app/docs/apim_yamles/yamles_externalized_files/

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

The changes aims to provide yamles end-user documentation on the new entities that have been externalized.

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

File: content/en/docs/apim_yamles/yamles_externalized_files.md [hunk 1]
```diff
@@ -55,18 +55,25 @@ During conversion from FED, externalized file names are generated as in the foll
...
 | --- | --- | --- | --- | --- | --- | --- |
-| Script language / Scripts (in Resources) | `JavaScriptFilter`, `Script` | `script#ref` | `<Parent entity>-Scripts/<entity field "name">` | `.groovy`, `.js`, `.nashorn.js`, `.py` |  | Yes |
-| Set Message (Policy Filter) | FilterCircuit | `body#ref` | `<Parent entity>-<entity field "name">` | `.json`, `.html`, `.xml`, `.txt` | Only multi-line body is externalized. Extension depends on content type content. | Yes |
+| Script language / Scripts (in Resources) | `JavaScriptFilter`, `Script` | `script#ref` | `<Parent entity>-Files/<entity field "name">` | `.groovy`, `.js`, `.nashorn.js`, `.py` |  | Yes |
+| Set Message (Policy Filter) | FilterCircuit | `body#ref` | `<Parent entity>-Files/<entity field "name">` | `.json`, `.html`, `.xml`, `.txt` | Only multi-line body is externalize. Extension depends on content type content. | Yes |
+| Attribute Extract Database  (Policy Filter) | FilterCircuit | `sqlStatement#ref` | `<Parent entity>-Files/<entity field "name">` | `.sql` | Only multi-line body is externalize. Extension depends on content type content. | No |
```
Comment (bordigoni) [inline content/en/docs/apim_yamles/yamles_externalized_files.md:60]: Remove "extension depends... ".

Comment (bordigoni) [inline content/en/docs/apim_yamles/yamles_externalized_files.md:60]: E18n is possible on non binary files. Unlikely but possible

**Review by bordigoni — COMMENTED** [review]
Good job at grouping the filters together. Just tiny errors on SQL to fix for me.

**Review by allandenis38 — APPROVED** [review]

File: content/en/docs/apim_yamles/yamles_externalized_files.md [hunk 2]
```diff
@@ -55,18 +55,25 @@ During conversion from FED, externalized file names are generated as in the foll
 
 | Entity | EntityType | Field | Externalized file name scheme | Possible extensions | Additional rules | Environmentalization possible inside file content |
 | --- | --- | --- | --- | --- | --- | --- |
-| Script language / Scripts (in Resources) | `JavaScriptFilter`, `Script` | `script#ref` | `<Parent entity>-Scripts/<entity field "name">` | `.groovy`, `.js`, `.nashorn.js`, `.py` |  | Yes |
-| Set Message (Policy Filter) | FilterCircuit | `body#ref` | `<Parent entity>-<entity field "name">` | `.json`, `.html`, `.xml`, `.txt` | Only multi-line body is externalized. Extension depends on content type content. | Yes |
+| Script language / Scripts (in Resources) | `JavaScriptFilter`, `Script` | `script#ref` | `<Parent entity>-Files/<entity field "name">` | `.groovy`, `.js`, `.nashorn.js`, `.py` |  | Yes |
```
Comment (andreamussap) [inline content/en/docs/apim_yamles/yamles_externalized_files.md:58]: @dominiquemongelli @allandenis38 why some EntityTypes are formatted as code, but others don't? Is this correct?

File: content/en/docs/apim_yamles/yamles_externalized_files.md [hunk 3]
```diff
@@ -55,18 +55,25 @@ During conversion from FED, externalized file names are generated as in the foll
...
-| Alert type → OPSEC → Files to upload | `UpLoadFile` | `contents#binary` | `upload-files/<entity fields "name">` | n/a | Entity name is the file name. | No |
-| PGP Key Pairs | PGPKeyPairs | `publicKey#binary`  | `<entity field "alias">-pub` | `.crt` |  | No |
-|               |             | `privateKey#binary` | `<entity field "alias">-sec` | `.asc` |  | No |
-| Tivoli Connection | TivoliSettings | `configDatabaseFile#binary` | `<entity field "name">` | `.db.conf`   | Tivoli Configuration Database File. | No |
-|                   |                | `sslStashFile#binary`       | `<entity field "name">` | `.ssl.key`   | SSL Stash file                      | No |
-|                   |                | `sslKeyFile#binary`         | `<entity field "name">` | `.ssl.stash` | SSL Key file                        | No |
+| Alert type → OPSEC   | `OpsecAlertSystem` | `connectionInfo#ref` | `<entity fields "name">` | `.conf` | Entity name is the file name. | No |
```
Comment (andreamussap) [inline content/en/docs/apim_yamles/yamles_externalized_files.md:65]: @dominiquemongelli  What's this arrow? Alert type → OPSEC 
Can this be changed to **>**. I'm afraid that the arrow won't convert well in some browsers, or on zoomin site...

**Comment by andreamussap** [comment]
@dominiquemongelli good job on adding the Jira ticket to the title of the PR, and naming your feature branch with a descriptive name. Just one more tip for next PRs, it helps if you add the deploy preview link, it makes it easier for Reviewers to review the changes (I've added the link to the description of the PR)

File: content/en/docs/apim_yamles/yamles_externalized_files.md (hunk 2 above)
Comment (dominiquemongelli) [reply content/en/docs/apim_yamles/yamles_externalized_files.md:58]: good point. I'll make sure all EntityTypes is formatted as code

File: content/en/docs/apim_yamles/yamles_externalized_files.md (hunk 3 above)
Comment (dominiquemongelli) [reply content/en/docs/apim_yamles/yamles_externalized_files.md:65]: No problem.

**Pull request merged**